app/controllers/summary_controller.rb

```diff
@@ -2,27 +2,40 @@ class SummaryController < ApplicationController
   before_action :authorize
 
   def index
-    client_namespace = current_user.client_slug.titleize
-    @client_summarizer = ClientSummarizer.new(
-      client_namespace: client_namespace,
-      fiscal_year: params[:fiscal_year]
-    )
-    @summary = @client_summarizer.run
+    @summaries = titleized_client_namespaces.map do |cn|
+      get_client_summary(cn, params[:fiscal_year])
+    end
   end
 
   private
 
   def authorize
-    if !adminish_role? || !client_slug?
+    if !current_user.any_admin? || (needs_client_slug? && !client_slug?)
```

Comment:
`client_admin` is only meant to see a summary of their own `client_slug` (i.e. the client for which they are an admin). Other admins are meant to see summaries of _all_ clients. So `titleize_client_namespaces` returns an array containing all the clients for which which this user is allowed to see summaries.